README.md

```diff
@@ -3,6 +3,12 @@
 The CSI-Addons project hosts extensions to the [CSI specification][csi_spec]
 that provide advanced storage operations.
 
+## Network Fencing
+
+The [Network Fencing](fencing/README.md) specification provides a
+mechanism that Storage Providers can implement to blocklist any
```

Comment:
please do not use (Ceph specific) blocklist here, call it network-fencing instead

Reply:
Thanks for the suggestion! Updated now :)